core/App/ehsm_provider.h

```diff
@@ -32,205 +32,313 @@
 #ifndef EHSM_RPOVIDER_H
 #define EHSM_PROVIDER_H
 
-#include "common_ehsm.h"
-
-typedef enum _ra_msg_type_t
-{
-     TYPE_RA_MSG0 = 0,
-     TYPE_RA_MSG1,
-     TYPE_RA_MSG2,
-     TYPE_RA_MSG3,
-     TYPE_RA_ATT_RESULT,
-     TYPE_RA_RETRIEVE_DK,
-}ra_msg_type_t;
+#include <stdint.h>
+
+#define _T(x) x
+
+#define ENCLAVE_PATH "libenclave-ehsm-core.signed.so"
```

Comment:
refer to https://github.com/intel/ehsm/commit/a8db0bc56f8dbf3f46d1e8b7003f6d27d794df2a

Reply:
will update it in next patch